airtable/plugin.py

```diff
@@ -30,16 +38,15 @@ class AirtableRecordID:
 
 def airtable_parse_ids_from_url(
     url: str,
-) -> Tuple[AirtableBaseID, Optional[AirtableTableID], Optional[AirtableRecordID]]:
+) -> Tuple[AirtableBaseID, Optional[AirtableTableID], Optional[AirtableRecordID], Optional[AirtableViewID]]:
```

Comment:
Order the tuple the same as how they appear in the URL?  I think keeping the order consistent is useful (at a minimum for human-reading purposes), and the URL order seems sensible.